tests_e2e/docker/Dockerfile

```diff
@@ -23,14 +23,20 @@ RUN \
     #                                                                                                            \
     # Install basic dependencies                                                                                 \
     #                                                                                                            \
-    apt-get install -y git python3.10 python3.10-dev curl                                                     && \
```

Comment:
curl was previously use to download Poetry in order to setup LISA, but their new installation is not based on Poetry so curl is no longer needed